langwatch/src/pages/api/health.ts

```diff
@@ -0,0 +1,8 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+
+export default function handler(
+  _req: NextApiRequest,
+  res: NextApiResponse
+) {
+  res.status(204).json({ status: 'ok' });
```

Comment:
Just need this too, as 204 can't have a body.

```suggestion
  res.status(204);
```